tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+# Initialization script responsible for setting up port forwarding for Istio sidecar.
+
+set -o nounset
+set -o pipefail
+IFS=,
+
+ISTIO_SIDECAR_CONFIG=${ISTIO_SIDECAR_CONFIG:-/var/lib/istio/envoy/sidecar.env}
+if [ ! -r $ISTIO_SIDECAR_CONFIG ]; then
+  echo "Missing sidecar config: $ISTIO_SIDECAR_CONFIG"
+  exit 1
+fi
+
+. $ISTIO_SIDECAR_CONFIG
+
+if [ -z "${ENVOY_UID:-}" ]; then
+  # Default to the UID of ENVOY_USER
+  ENVOY_UID=$(id -u ${ENVOY_USER:-istio})
+  if [ ! $? == 0 ]; then
+     echo "Invalid istio user $ENVOY_UID $ENVOY_USER"
+     exit 1
+  fi
+  ENVOY_UID=${ENVOY_UID},0
```

Comment:
rename to PROTECTED_UIDS or UNINTERCEPTED_UIDS here and where used maybe ? (would be clearer)